src/OpenApi/src/Extensions/JsonNodeSchemaExtensions.cs

```diff
@@ -365,7 +363,8 @@ internal static void ApplyNullabilityContextInfo(this JsonNode schema, Parameter
             return;
         }
 
-        var nullabilityInfo = _nullabilityInfoContext.Create(parameterInfo);
+        var nullabilityInfoContext = new NullabilityInfoContext();
```

Comment:
You can use JsonPropertyInfo.AssociatedParameterInfo.IsNullable here.

Reply:
It looks like that API is limited to properties that are sourced from a constructor parameter? This particular case applies to parameters source from the handler method, such as `name` in`app.MapGet("/", (string name) => ...)`. I don't think there's a JSON info-based API for that?

Reply:
Ah yes, you're correct. You're on your own when it comes to method parameters.

Reply:
One thing you might want to be aware of is that `NullabilityInfoContext` has a feature switch that can turn it off (resulting in exceptions). We learned this the hard way because the blazor sdk was turning it off by default, so I had to introduce defensive code in the STJ side that [checked for the feature switch](https://github.com/dotnet/runtime/pull/102852) before exercising `NullabilityInfoContext` APIs.

We decided in API review that the feature switch was a bad idea and we're going to remove it [at some point](https://github.com/dotnet/runtime/issues/103298#issuecomment-2161338112), though I'm not sure we'll be able to get to that for .NET 9.

Reply:
> One thing you might want to be aware of is that NullabilityInfoContext has a feature switch that can turn it off (resulting in exceptions being thrown every time you call its APIs). We learned this the hard way because the blazor sdk was turning it off by default, so I had to introduce defensive code in the STJ side that https://github.com/dotnet/runtime/pull/102852 before exercising NullabilityInfoContext APIs.

Good note!  I've applied a similar check here to guard against this.

Reply:
I just deleted the feature switch altogether from runtime, so assuming you only target .NET 9 you should be good.

Reply:
Yep, I went ahead and remove this check from here as well.